lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -5190,8 +5285,11 @@ ParseResult FIRCircuitParser::parseLayer(CircuitOp circuit) {
     StringRef id, convention;
     LocWithInfo info(getToken().getLoc(), this);
     consumeToken();
-    if (parseId(id, "expected layer name") || parseGetSpelling(convention))
+    if (parseId(id, "expected layer name") ||
+        parseToken(FIRToken::comma, "expected ','") ||
```

Comment:
Is a comma here in the spec?  It looks a little confusing and I don't know that we should add it if it wasn't already.

Reply:
The spec doesn't have it. The tests had it both ways.

I decided to require the comma here. I'll update the spec accordingly after this change goes through.